cmd/bnsd/app/app_test.go

```diff
@@ -161,6 +164,10 @@ func TestApp(t *testing.T) {
 func tagsAsString(pairs []common.KVPair) string {
 	r := make([]string, len(pairs))
 	for i, v := range pairs {
+		if string(v.Key) == "action" {
```

Comment:
q: I stumbled upon this lines as we have a mix of encoded and unencoded tag keys now. Would this be a problem for the clients? How can we document the "default" tags the best way for any consumers?

Reply:
Consumers (iov-core) do not parse out all tags into string form to compare. I do this just to test it works. They create a specific tag of interest and use that to search/subscribe. This will not interfere, just add another way of searching. (Which should be documented well enough in the CHANGELOG for them to implement :pray: )